frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/review-and-create-step.tsx

```diff
@@ -17,19 +17,21 @@ import {
   NodesCard,
   RequestErrors,
 } from '../../../install-wizard/review-and-create';
-import { NetworkType } from '../../../types';
+import { NetworkType, SupportedFeatures } from '../../../types';
 
 export const ReviewAndCreate: React.FC<ReviewAndCreateProps> = ({
   state,
   errorMessage,
   inProgress,
+  supportedFeatures,
 }) => {
   const { t } = useTranslation();
 
   const { nodes, encryption, enableMinimal, storageClass, kms, networkType, publicNetwork } = state;
   const { cpu, memory, zones } = getNodeInfo(state.nodes);
   const scName = getName(storageClass);
   const emptyRequiredField = nodes.length < MINIMUM_NODES && !scName && !memory && !cpu;
+  const isMultusSupported = supportedFeatures?.includes(SupportedFeatures.MULTUS);
```

Comment:
Lets move this to the parent component, and pass down to children.

Reply:
I don't agree with this architecture pages like these can have multiple features which get changed over time, passing this via parent is just adding a lot of props.

Reply:
The logic is recalculated in each component, hence moving up makes sense

Reply:
We can calculate each in parent and pass the values as flags.